ui/components/ui/account-list/account-list.js

```diff
@@ -61,7 +63,11 @@ const AccountList = ({
                 </div>
               }
             >
-              <i className="fa fa-info-circle" />
+              <Icon
+                name={ICON_NAMES.INFO}
+                color={IconColor.iconMuted}
+                className="info-circle"
```

Comment:
Is this class necessary? Can you use box props instead?

Reply:
I think we need it for hover